tidy3d/components/medium.py

```diff
@@ -71,6 +71,53 @@ def _eps_model(self, frequency: float) -> complex:
 """ Medium Definitions """
 
 
+class Chi3Spec(Tidy3dBaseModel):
```

Comment:
Do we want an abstract `NonlinearSpec`? or I guess maybe later?

Reply:
maybe later I think